doc/whats_new.rst

```diff
@@ -251,6 +252,12 @@ API changes summary
       :func:`sklearn.model_selection.cross_val_predict`.
       :issue:`2879` by :user:`Stephen Hoover <stephen-hoover>`.
 
+    - In :class:`pipeline.Pipeline`, ``steps`` has been changed by ``steps`` and
```

Comment:
To be sure, are you suggesting to replace the `FutureWarning` with a `DeprecationWarning` mentioning that the behaviour will change in 0.21 (or 0.22)